modules/fluid_properties/test/tests/functions/saturation_temperature_function/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  [./test]
+    type = 'CSVDiff'
+    input = 'saturation_temperature_function.i'
+    csvdiff = 'saturation_temperature_function_out.csv'
+    allow_test_objects = True
+
+    requirement = 'MOOSE shall provide a function that computes saturation temperature from a pressure function and 2-phase fluid properties object'
```

Comment:
@permcody Should this be "The system shall..."? Did we settle on the proper wording yet?

Reply:
Andrew said the preferred wording is "The system shall", so I'll change it.